action.yml

```diff
@@ -29,6 +29,9 @@ inputs:
     description: 'Switch to this builder instance'
     default: 'true'
     required: false
+  endpoint:
+    description: 'Address for docker socket or context from `docker context ls (eg. DOCKER_HOST value)'
```

Comment:
missing closing `

Maybe also "Optional address for remote docker socket or..."

Reply:
Yeah "optional address" looks better.